target_s3_csv/__init__.py

```diff
@@ -151,6 +151,11 @@ def persist_messages(messages, config, s3_client):
             schema = utils.float_to_decimal(o['schema'])
             validators[stream] = Draft7Validator(schema, format_checker=FormatChecker())
             key_properties[stream] = o['key_properties']
+            
+            filename = o['stream'] + '-schema.json'
+            with open(filename, 'w+') as f:
+                f.write(o['stream'])
```

Comment:
Should be 
```python
f.write(o['schema'])
```

Reply:
I thought the type was SCHEMA and the data was in stream will update

Reply:
or actually just 
```python
f.write(schema) 
```
to make sure you have the metadata columns included